source/features/show-names.tsx

```diff
@@ -5,6 +5,13 @@ import * as api from '../libs/api';
 import features from '../libs/features';
 import {getUsername} from '../libs/utils';
 
+// Returns true if username is basiclly the same as realname
+function compareNames(username: string, realname: string): boolean {
+	const reg = /[ .]/g;
+
+	return username.replace(reg, '').toLowerCase() === realname.replace(reg, '').toLowerCase();
```

Comment:
Checking the username for spaces is not necessary, as per GitHub self:
> Username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen.

Reply:
```suggestion
	return username.replace(/[.-]/g, '').toLowerCase() === realname.replace(/[^W]/gi, '').toLowerCase();
```

But the comparison needs to be even better: `@nicolo` won't match `Nicolò` without unicode normalization.

Reply:
`String.prototype.noramlize` is available for unicode normalization:

```
realname.normalize(‘NFD’).replace(/[\u0300-\u036f]/g, '').replace(/[^W]/giu, '').toLowerCase();
```

Reply:
When you implement this, it would be good to have tests to match usernames to their real names, to ensure it works correctly without having to find them in the real world.